be/src/exec/hdfs_scanner.cpp

```diff
@@ -578,6 +581,29 @@ void HdfsScannerContext::append_or_update_partition_column_to_chunk(ChunkPtr* ch
     ck->set_num_rows(row_count);
 }
 
+void HdfsScannerContext::append_or_update_extended_column_to_chunk(ChunkPtr* chunk, size_t row_count) {
```

Comment:
If this is the same as the partition column, why not merge them and make it more general

Reply:
Yes, please consider to merge into that function,  it will be less error-prone.

Reply:
+1

Reply:
done

Reply:
didn't see you change anything?

Reply:
this commit https://github.com/StarRocks/starrocks/pull/51050/commits/b918629ff3086449aa68a4c90b6b33785bb7f279